### DISCUSSION TITLE
Inheritance and instancing

### DISCUSSION THREAD

**Discussion by RinoGuideDog** [jmmorato/openddsharp#245] [Q&A]
I Can instance a struct into another but if I have multiple instances on one namespace it has a bit of a fit, any suggestions on what to do / methods I can have different namespaces on the same project file? Also is inheritances a  thing in this yet I've tried     
"struct data: Dataheader {
        Signaldata Signaldata;
    };"
to see and I'm unsure if I've just done it wrong or that  its not supported.

**Comment by jmmorato** [comment]
Not sure that I understand your issue, can you share your IDL?
